buildSrc/src/main/kotlin/com/datadog/gradle/config/PublishingConfig.kt

```diff
@@ -41,18 +41,26 @@ fun Project.publishingConfig(
                 artifactId = projectName
                 version = AndroidConfig.VERSION.name
 
-                artifact("$buildDir/outputs/aar/$projectName-release.aar")
+                if (asAar) {
```

Comment:
I think you need to rebase as these changes should not be part of this commit.

Reply:
I rebased off of master, I think this one was one of the conflicts that I resolved earlier. Should this change not be on the master branch?

Reply:
Nope...this line is not in the `master`, I think this change was part of a previous state of master when we had the `datadog-gradle-plugin` package inside. @xgouchet correct me if I am wrong.